### ISSUE TITLE
Unresolved reference to 'targets' in transfer_learning_chatbot.py

### ISSUE THREAD

**Issue by AlanDoesCS** [adrirubio/Mindful#8]
# Describe the issue
In `transfer_learning_chatbot.py`, there are four occurrences of `targets` in the accuracy calculation, though it is never defined

```python
# ...
# Accuracy
n_correct = 0
n_total = 0
for batch in train_loader:
    input_ids, labels = batch
    input_ids = input_ids.to(device)
    labels = labels.to(device)

    # Forward pass
    outputs = model(input_ids=input_ids, labels=labels)

    # Get predictions
    _, predictions = torch.max(outputs, 1)

    # Update counts
    n_correct += (predictions == targets).sum().item()  # <-- here
    n_total += targets.shape[0]  # <-- here

train_acc = n_correct / n_total

n_correct = 0
n_total = 0
for batch in test_loader:
    input_ids, labels = batch
    input_ids = input_ids.to(device)
    labels = labels.to(device)

    # Forward pass
    outputs = model(input_ids=input_ids, labels=labels)

    # Get predictions
    _, predictions = torch.max(outputs, 1)

    # Update counts
    n_correct += (predictions == targets).sum().item()  # <-- here
    n_total += targets.shape[0]  # <-- here

test_acc = n_correct / n_total
# ...
```

**Comment by adrirubio** [comment]
Good point, I already fixed the error!

**Comment by AlanDoesCS** [comment]
Thanks Adrian!
